components/x-teaser/src/v2/components/Meta/styles.scss

```diff
@@ -0,0 +1,69 @@
+@import '@financial-times/o-typography/main';
+@import '../../themes/themes';
+
+$space-between-labels: 5px;
+
+.meta {
+	@include oTypographySize($scale: 0);
+
+	a {
+		color: inherit;
+		text-decoration: none;
+	}
+
+	&:after {
+		margin-top: 0;
+		content: '';
+		display: block;
+		width: 60px;
+		border-bottom-width: 0;
+		border-bottom-style: solid;
+	}
+
+	@include themify {
+		color: themed('metaColorPrimary');
+
+		a {
+			&:hover,
+			&:focus {
+				color: themed('metaColorHover');
+			}
+		}
+
+		&:after {
+			border-bottom-color: themed('metaColorPrimary');
+		}
+	}
+}
+
+.hasBorder:after {
+	border-bottom-width: oTypographySpacingSize(1);
+	margin-top: oTypographySpacingSize(2);
```

Comment:
This is all good, but [o-spacing](https://github.com/Financial-Times/o-spacing) is available now 🎉 
`oTypographySpacingSize(2)` -> `oSpacingByName('s2')`

Reply:
updated